app/src/main/java/kr/co/connect/boostcamp/livewhere/ui/map/adapter/MapMarkerAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package kr.co.connect.boostcamp.livewhere.ui.map.adapter
+
+import android.content.Context
+import android.view.View
+import com.naver.maps.map.overlay.InfoWindow
+import kotlinx.android.synthetic.main.dialog_window_map_info.view.*
+import kr.co.connect.boostcamp.livewhere.R
+
+//마커 위의 글씨를 보여주는 info
+class MapMarkerAdapter(private val mContext: Context, private val title: String) :
```

Comment:
```suggestion
class MapMarkerAdapter(private val context: Context, private val title: String) :
```
P2. 맴버변수를 뜻하는 m은 제거해주시면 좋을 것 같습니다.

Reply:
P1. 만약 인자로 전달받은 Context가 유효하지 않을 경우에는 문제가 발생할 요지가 다분합니다.
어떤 경우에 Context가 유효하지 않으며 이를 해결하기 위해서는 어떻게 하면 좋을까요?